README.md

```diff
@@ -196,10 +196,10 @@ and
 :FzfPreviewCommandPalette                         " Execute and edit command history
 :CocCommand fzf-preview.CommandPalette
 
-:FzfPreviewGitActions                         " Interactive git integration. (Required [Fugitive](https://github.com/tpope/vim-fugitive) and [Gina](https://github.com/lambdalisue/gina.vim))
+:FzfPreviewGitActions                         " Interactive git integration. (Required [Fugitive](https://github.com/tpope/vim-fugitive) or [Gina](https://github.com/lambdalisue/gina.vim))
```

Comment:
this should be `or`, I get confused at first time.

The other sentences are written `or` as well.

https://github.com/yuki-ycino/fzf-preview.vim/blob/9719e67fdcd9dc8f938d4398a22ec5f4aca1d875/README.md#L30